lib/phoenix/digester.ex

```diff
@@ -144,10 +150,14 @@ defmodule Phoenix.Digester do
 
         case Map.fetch(manifest, manifest_path) do
           {:ok, digested_path} ->
-            url
+            url = url
```

Comment:
```elixir
url =
  url
  |> Path.join(Path.basename(digested_path))
  |> Path.dirname()
```
Otherwise it's very unclear where first expression ends.